packages/core/bootstrap/test/unit/rate-limit-cache-tests/1-rate-limit-cache.test.ts

```diff
@@ -184,8 +184,8 @@ describe('Rate Limit/Cache - Integration', () => {
     const rlPerMinute = getRLTokenSpentPerMinute(state.rateLimit.heartbeats)
 
     Object.values(rlPerMinute).forEach((req) => {
-      // TODO: check that + 30 is the right capacity
-      expect(req).toBeLessThan(capacity + 30)
```

Comment:
Should this change be a part of this PR?

Reply:
It doesn't necessarily have to be a part of this, but this unit test failed at one point since `req` was `81` when `capacity + 30` was `80`. Probably worth investigating in its own right.